common/src/main/java/dev/latvian/kubejs/script/data/DataPackEventJS.java

```diff
@@ -10,6 +12,14 @@
 /**
  * @author LatvianModder
  */
+@KubeJSEvent(
+		server = {
+				KubeJSEvents.SERVER_DATAPACK_LOW_PRIORITY,
+				KubeJSEvents.SERVER_DATAPACK_HIGH_PRIORITY,
+				"server.datapack.first",
```

Comment:
Shouldn't these be changed to the event IDs you've added to KubeJSEvents?

Reply:
Fixed in the latest commit.